### PR TITLE
Cleanup .gitignore

### PR DESCRIPTION
**Pull request by electrum**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,4 +8,3 @@ wasi-testsuite/
 dependency-reduced-pom.xml
 jmh-result.json
 scripts/jmh-tmp
-main/
```
Comment (andreaTP) [inline .gitignore:11]: I agree the naming is not ideal but this is the folder where the baseline chicory repository gets cloned for jmh tests, I would keep it in `gitignore`

**Review by andreaTP — COMMENTED** [review]
I think we should keep `main`, apart from that, good to go 👍 thanks!

File: .gitignore (hunk 1 above)
Comment (andreaTP) [reply .gitignore:11]: it should be `/main/` and is fixed in the `main` branch already.

**Comment by andreaTP** [comment]
@electrum please rebase and it's good to go, thanks 👍

**Comment by electrum** [comment]
I see you changed `main/` to `/main/` so this is good. The previous one was blocking `src/main/java`.

**Comment by electrum** [comment]
I rebased and dropped the second commit.

**Pull request merged**